### DISCUSSION TITLE
Harvester

### DISCUSSION THREAD

**Discussion by mattmattox** [selfhst/icons#151] [New Icon Request]
It would be cool if we could add the Rancher's project icons
https://www.rancher.com/brand-guidelines

NOTE: Rancher's icons are already in the repo, this request is just to add Harvester, RKE, K3s, Longhorn, etc.

**Comment by shollyethan** [comment]
Thanks for the suggestion, these have been added! Note that all of the actual icon reference names are prefaced with 'rancher-' to easily allow searching for them and to prevent the more generic names from conflicting with others.
